ads/smartadserver.md

```diff
@@ -16,14 +16,29 @@ limitations under the License.
 
 # SmartAdServer
 
+## Supported Smart AdServer parameters in the amp-ad tag
+
+All of the parameters listed here should be prefixed with "data-" when used.
+
+| Parameter name | Description                         | Required |
+|----------------|-------------------------------------|----------|
+| site           | Your Smart AdServer Site ID         | Yes      |
+| page           | Your Smart AdServer Page ID         | Yes      |
+| format         | Your Smart AdServer Format ID       | Yes      |
+| domain         | Your Smart AdServer call domain     | Yes      |
+| target         | Your targeting string               | No       |
+| tag            | An ID for the tag containing the ad | No       |
+
+Note: If any of the required parameters is missing, the integration won't be able to work correctly,
+and a "noContentAvailable" notification will be returned by the Smart AdServer integration.
```

Comment:
We do call the ```window.context.noContentAvailable()``` function when appropriate, either before sending the request to the server if we absolutely know it can't be fulfilled, or in the response from the server.

As for ```renderStart```, it was not in our top priority list, but I will experiment a bit and see if I can get it to work reasonably in our case.